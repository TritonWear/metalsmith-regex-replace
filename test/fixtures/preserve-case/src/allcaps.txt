The lion said "RAWR I'M A LION!"