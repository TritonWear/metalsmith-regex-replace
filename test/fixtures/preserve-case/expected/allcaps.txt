The tiger said "RAWR I'M A TIGER!"